Makefile

```diff
@@ -88,48 +88,48 @@ art/mgmt_logo_white_wide.png: art/mgmt_logo_white_wide.svg
 all: docs $(PROGRAM).static
 
 # show the current version
-version:
+version: ## show the current version
 	@echo $(VERSION)
 
-program:
+program: ## show the program name
 	@echo $(PROGRAM)
 
-path:
+path: ## create working paths
 	./misc/make-path.sh
 
-deps:
+deps: ## install system and golang dependencies
 	./misc/make-deps.sh
 
-run:
+run: ## run mgmt
 	find . -maxdepth 1 -type f -name '*.go' -not -name '*_test.go' | xargs go run -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION)"
 
 # include race flag
 race:
 	find . -maxdepth 1 -type f -name '*.go' -not -name '*_test.go' | xargs go run -race -ldflags "-X main.program=$(PROGRAM) -X main.version=$(SVERSION)"
 
 # generate go files from non-go source
-bindata:
+bindata: ## generate go files from non-go sources
 	@echo "Generating: bindata..."
 	$(MAKE) --quiet -C bindata
 
 generate:
 	go generate
 
-lang:
+lang: ## generates the lexer/parser for the language frontend
 	@# recursively run make in child dir named lang
 	@echo "Generating: lang..."
 	$(MAKE) --quiet -C lang
 
 # build a `mgmt` binary for current host os/arch
-$(PROGRAM): build/mgmt-${GOHOSTOS}-${GOHOSTARCH}
+$(PROGRAM): build/mgmt-${GOHOSTOS}-${GOHOSTARCH} ## build a mgmt binary for current host os/arch
```

Comment:
build a mgmt binary for current host os/arch  ->  build an mgmt binary for current host os/arch

please :) a -> an

Reply:
Fixed. How is `mgmt` pronounced? I thought it was 'management' or is it 'em jee em tee'?